diagrams/resources/web-framework-diagram.drawio

```diff
@@ -124,6 +100,33 @@
         <mxCell id="7qe8jE41nwvI0GSgo9u9-10" value="&lt;font style=&quot;font-size: 22px; color: rgb(255, 255, 255);&quot;&gt;Accessibility&lt;/font&gt;" style="text;html=1;align=center;verticalAlign=middle;whiteSpace=wrap;rounded=0;" parent="1" vertex="1">
           <mxGeometry x="745" y="415" width="60" height="30" as="geometry" />
         </mxCell>
+        <mxCell id="NK9HXle7FpP0BrMxsV12-2" value="&lt;font style=&quot;font-size: 22px; color: rgb(255, 255, 255);&quot;&gt;WebGL&lt;/font&gt;" style="text;html=1;align=center;verticalAlign=middle;whiteSpace=wrap;rounded=0;" parent="1" vertex="1">
+          <mxGeometry x="584.5" y="489" width="60" height="30" as="geometry" />
+        </mxCell>
+        <mxCell id="uTWjtHHHEZxcMKsox1Cd-2" value="&lt;font style=&quot;font-size: 22px;&quot;&gt;&lt;b&gt;Browser&lt;/b&gt;&lt;/font&gt;&lt;div&gt;&lt;br&gt;&lt;/div&gt;" style="text;html=1;align=left;verticalAlign=middle;whiteSpace=wrap;rounded=0;" vertex="1" parent="1">
+          <mxGeometry x="30" y="494" width="150" height="30" as="geometry" />
+        </mxCell>
+        <mxCell id="uTWjtHHHEZxcMKsox1Cd-4" value="&lt;font style=&quot;font-size: 22px; color: rgb(255, 255, 255);&quot;&gt;HTML/CSS&lt;/font&gt;" style="text;html=1;align=center;verticalAlign=middle;whiteSpace=wrap;rounded=0;" vertex="1" parent="1">
```

Comment:
I STILL think we should drop `HTML/CSS` from this box. I think it's more confusing than helpful. 🤷 

@yjbanov ?

Reply:
He just had me add it back.